### PR TITLE
Updates for changes to MooseDocs

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #3)

### CODE REVIEW

**Comment by aeslaughter** [comment]
@sveerara This should help get the website back into a better state. Merge at your will.

File: doc/content/manuals/theory/index.md [hunk 1]
```diff
@@ -148,17 +156,17 @@ To model the mechanical behavior of a material, three components are need to be
 Details about stress calculation for two different constitutive models are presented below.
```
Comment (sveerara) [inline doc/content/manuals/theory/index.md]: In the strain section above I have added a link to the moose tensor mechanics documentation from a PR that was open during the SQA meeting. Can this link be replaced with a better link that shows the current version of the Tensor mechanics documentation?

**Comment by sveerara** [comment]
@aeslaughter, thanks for working on the website. The commits look good.

I don't have merge permissions for this repository.

**Comment by aeslaughter** [comment]
@sveerara You should have access to merge now.

**Pull request merged**